pkg/scheduler/api/node_info.go

```diff
@@ -263,22 +263,33 @@ func (ni *NodeInfo) setOversubscription(node *v1.Node) {
 	}
 }
 
+// SetOutOfSync sets state of nodeInfo to `OutOfSync`
+func (ni *NodeInfo) SetOutOfSync() {
+	state := NodeState{
+		Phase:  NotReady,
+		Reason: ReasonOutOfSync,
+	}
+	ni.State = state
+}
+
+// OutOfSync determines if state of nodeInfo is `OutOfSync`
+func (ni *NodeInfo) OutOfSync() bool {
```

Comment:
Suggest to follow the common code of conduct. For this function is to judge the node status, perhaps name it as `IsOutOfSync` is better.